src/plone/api/relation.py

```diff
@@ -140,12 +140,10 @@ def create(source=None, target=None, relationship=None):
     :type source: Content object
     :param target: [required] Object that the relation will point to.
     :type target: Content object
-    :param relationship: [required] Relationship name. If that name is the same
-    as a field name and this field is a RelationChoice/RelationList we will
-    update the field-value accordingly.
+    :param relationship: [required] Relationship name. If that name is the same as a field name and this field is a RelationChoice / RelationList we will update the field-value accordingly.
```

Comment:
What is the style that maintainers follow for line wrapping in docstrings?

Reply:
One line is not nice. It's a bug fix. As soon as the string is splitted in multiple lines, the autodoc fails. This is of course because the description is not expected to be so long. Authors can review later.

Reply:
I think you can have multiline docstrings for params if they are indented.

https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#overview

```suggestion
    :param relationship: [required] Relationship name.
        If that name is the same as a field name and this field is a
        RelationChoice / RelationList we will update the field-value
        accordingly.
```